.github/workflows/scp-items.yml

```diff
@@ -0,0 +1,92 @@
+name: Crawl SCP Wiki
+
+on:
+#   workflow_dispatch:
+#   schedule:
+#     - cron: "0 0 * * *"
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/scp-items.yml
```

Comment:
Lets have this run on PRs too so we can see when people make changes that break things.